pandas/tests/extension/base/setitem.py

```diff
@@ -32,22 +34,25 @@ def test_setitem_sequence_mismatched_length_raises(self, data, as_array):
         xpr = 'cannot set using a {} indexer with a different length'
         with tm.assert_raises_regex(ValueError, xpr.format('list-like')):
             ser[[0, 1]] = value
+        assert ser._values[[0, 1]] == value
 
         with tm.assert_raises_regex(ValueError, xpr.format('slice')):
             ser[slice(3)] = value
-
-    def test_setitem_empty_indxer(self, data):
-        ser = pd.Series(data)
-        original = ser.copy()
-        ser[[]] = []
-        self.assert_series_equal(ser, original)
-
-    def test_setitem_sequence_broadcasts(self, data):
-        arr = pd.Series(data)
-
-        arr[[0, 1]] = data[2]
-        assert arr[0] == data[2]
-        assert arr[1] == data[2]
+        assert ser._values[slice(3)] == value
+
+    def test_setitem_empty_indxer(self, data, box_in_series):
+        if box_in_series:
```

Comment:
The indentation is messed up here. This test is basically skipped when box_in_series is False.

You should dedent on line 47 and change the assert to `self.assert_equal`.

Reply:
changed... appreciate!! and happy to learn a lot about tests... ^^